ckanext/recombinant/write_excel.py

```diff
@@ -226,10 +226,13 @@ def excel_data_dictionary(geno):
                 _append_field_ref_rows(refs, field, link=None)
 
                 if field['datastore_id'] in choice_fields:
+                    full_text_choices = (
+                        field['datastore_type'] != '_text' and field.get(
+                        'excel_full_text_choices', False))
```

Comment:
this looks wrong. `full_text_choices` is for when we want the choice drop-down to include both the code and the text.

Reply:
This is for the fix to the Reference sheet, where the `full_text_choices` where in column A instead of spanning column A and B

Reply:
oh I read this wrong. Is there a reason we need to check `field['datastore_type'] != '_text'`? Seems like we shouldn't have `excel_full_text_choices` set for any of those fields